tools/templates/nova_lfe/app.src

```diff
@@ -0,0 +1,19 @@
+%% -*- mode: erlang;erlang-indent-level: 4;indent-tabs-mode: nil -*-
+{application, {{name}},
+ [{description, "{{name}} managed by Nova"},
+  {vsn, "0.1.0"},
+  {registered, []},
+  {mod, { '{{name}}.app', []}},
+  {included_applications, []},
+  {applications,
+   [
+    kernel,
+    stdlib,
+    nova
+   ]},
+  {env,[]},
+  {modules, []},
+  {maintainers, []},
+  {licenses, ["Apache 2.0"]},
+  {links, []}
+ ]}.
```

Comment:
Isn't this almost the same as for the regular app.src? Just thinking that we could use that one instead